src/components/CreateOrJoin/CreateOrJoin.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import getToken from '../../lib/tokens';
+import { db } from '../../lib/firebase';
+
+const CreateOrJoin = (props) => {
+  const storeToken = () => {
+    const token = getToken();
+    props.updateToken(token);
+    db.collection(token).doc('ListData').set({ listCreated: new Date() });
+  };
+  return (
+    <div>
+      <h2>Welcome to your smart shopping list!</h2>
```

Comment:
This should be an h1 since we are removing the "Shopping app" title. Once you get to styling, you can use `<Typography variant="h1">` from MUI to make it use the theme font.

Reply:
thanks! I fixed this in the current styling branch I'm working on, so I'll go ahead and resolve these comments for now, but good call!